src/Bicep.Decompiler/TemplateConverter.cs

```diff
@@ -480,7 +499,19 @@ private SyntaxBase ParseFunctionExpression(FunctionExpression expression)
                         }
 
                         var stringVal = jTokenExpression.Value.Value<string>()!;
-                        var resolved = nameResolver.TryLookupName(NameType.Variable, stringVal) ?? throw new ArgumentException($"Unable to find variable {stringVal}");
+                        var resolved = nameResolver.TryLookupName(NameType.Variable, stringVal);
+                        if (resolved is null)
+                        {
+                            if (options.AllowMissingParamsAndVars)
+                            {
+                                resolved = nameResolver.TryRequestName(NameType.Parameter, stringVal)
+                                ?? throw new ArgumentException($"Unable to pick unique name for missing variable {stringVal}");
```

Comment:
We show this error message in Bicep Operations output channel. Shouldn't this be localized?

Reply:
Nothing here is localized.  I figure it will be done as a single pass when needed.